src/components/board/board.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './board.css';
+import Square from '../square/square';
+const boardFunction = require('../../services/boardService');
+const boardData = require('../../data');
+
+const board = boardFunction.makeBoard(boardData.FILES,boardData.RANKS);
```

Comment:
Move this inside the function before the return. It will help with the tests

Reply:
ok so Im getting an error when I move them inside the function, I switched the wording to import like the previous comment suggested, do I need to change it back to a const now?

Reply:
oh shoot - I wasn't getting an error. It helped with the test when I brought that call inside the function. It might not matter. Here's what I have:

```
import React from 'react';
import './board.css';
import Square from '../square/square';
const boardFunction = require('../../services/boardService');
const boardData = require('../../data');

export default function Board() {
  const board = boardFunction.makeBoard(boardData.FILES,boardData.RANKS);
  return (
    <div className='wrapper'>
      <div className='top'>
        <div className='wrapper-inner'>
        {boardData.FILES.map(value => ( <div key={value} className='box-inner'>{value}</div> ))}
        </div>
      </div>
      {board.map(value => (
        <Square value={value.coordinate} />
      ))}
      <div className='bottom'>
        <div className='wrapper-inner'>
        {boardData.FILES.map(value => ( <div key={value} className='box-inner'>{value}</div> ))}
        </div>
      </div>
    </div>
  );
}
``` 

it's more important that it works :-)